### PR TITLE
Raven db 3979

### PR DESCRIPTION
**Pull request by hilaMef**
https://github.com/ayende/ravendb/compare/v3.5...hilaMef:RavenDB-3979?expand=1

### CODE REVIEW

**Comment by ppekrol** [comment]
@hilaMef can you convert tabs to spaces in all your changed files?

File: Raven.Database/Util/DevelopmentHelper.cs [hunk 1]
```diff
@@ -13,7 +13,11 @@ internal static class DevelopmentHelper
     {
         public static void TimeBomb()
         {
+<<<<<<< HEAD
```
Comment (ppekrol) [inline Raven.Database/Util/DevelopmentHelper.cs]: This conflict needs to be resolved

**Comment by ayende** [comment]
Code doesn't compile.
Please fix this, as well as match the directions discussed in: https://groups.google.com/forum/#!topic/ravendb-internal/Q9WaUrzWveU

**Comment by hilaMef** [comment]
Im not a member of this group, please add me to it

**Pull request closed without merging**